Scripts/Diagnostics/Azure_Bastion_AuthZ_Disable_Shareable_Link.ps1

```diff
@@ -0,0 +1,145 @@
+$debug = $true
+
+function Update-BastionDisableShareableLink() {
+  <#
+  .SYNOPSIS
+  This function updates an Azure Bastion host to set the ShareableLink feature to the specified state.
+  .DESCRIPTION
+  This function updates an Azure Bastion host to set the ShareableLink feature to the specified state.
+  .PARAMETER SubscriptionId
+  The Azure subscription ID containing the Bastion host.
+  .PARAMETER ResourceGroupName
+  The Resource Group name containing the Bastion host.
+  .PARAMETER BastionHostName
+  The Bastion host name.
+  .PARAMETER ShareableLinkEnabled
+  Boolean for whether shareable link should be enabled. Defaults to false.
+  .INPUTS
+  None
+  .OUTPUTS
+  None
+  .EXAMPLE
+  PS> Update-BastionDisableShareableLink -SubscriptionId "00000000-xxxx-0000-xxxx-000000000000" -ResourceGroupName "MyResourceGroupName" -BastionHostName "MyKBastionHostName" -ShareableLinkEnabled $false
+  .LINK
+  None
+  #>
+
+  [CmdletBinding()]
+  param
+  (
+    [Parameter(Mandatory = $true)]
+    [string]
+    $SubscriptionId,
+    [Parameter(Mandatory = $true)]
+    [string]
+    $ResourceGroupName,
+    [Parameter(Mandatory = $true)]
+    [string]
+    $BastionHostName,
+    [Parameter(Mandatory = $false)]
+    [bool]
+    $ShareableLinkEnabled = $false
+  )
+
+  $profile = Set-AzContext -Subscription $SubscriptionId
```

Comment:
What if profile is null, shall we add docs to do connect-az account first?

Reply:
Readme has instruction to do connect-azaccount first